docs/src/main/sphinx/admin/properties-write-partitioning.rst

```diff
@@ -12,15 +12,18 @@ minimum number of partitions, set in ``preferred-write-partitioning-min-number-o
 are written, each partition is written by a separate writer. As a result, for some connectors such as the
 Hive connector, only a single new file is written per partition, instead of
 multiple files. Partition writer assignments are distributed across worker
-nodes for parallel processing.
+nodes for parallel processing. This can be specified on a per-query basis using
+the ``use_preferred_write_partitioning`` session property.
 
 ``preferred-write-partitioning-min-number-of-partitions``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 * **Type:** ``integer``
 * **Default value:** ``50``
 
-The minimum number of written partitions that is required to use connector
-``preferred write partitioning``. If the number of partitions cannot be
-estimated from the statistics, then preferred write partitioning is not used.
+Use connector ``preferred write partitioning`` when the optimizer's estimate
```

Comment:
This should not be in backquotes. It is not a property name (use_preferred_write_partitioning is the property name)

Reply:
updated